app/services/gitlab.js

```diff
@@ -0,0 +1,50 @@
+// https://gitlab.com/api/v4/projects/coala%2Fmobans/issues?order_by=created_at&state=opened
+import AjaxService from 'ember-ajax/services/ajax';
+import RSVP from 'rsvp';
+
+const ENDPOINT = 'https://gitlab.com/api/v4';
+const PROJECT_ENDPOINT = `${ENDPOINT}/projects/{projectId}`;
+const ISSUE_ENDPOINT = `${PROJECT_ENDPOINT}/issues?order_by=created_at&state=opened&per_page=100`;
+
+function buildIssuesUrl(projectId) {
+  return ISSUE_ENDPOINT.replace('{projectId}', encodeURIComponent(projectId));
+}
+
+function buildProjectUrl(projectId) {
+  return PROJECT_ENDPOINT.replace('{projectId}', encodeURIComponent(projectId));
+}
+
+export default AjaxService.extend({
+  host: 'https://gitlab.com/',
+  _issueUrl: '',
+  init(...arg) {
+    this._super(...arg);
+    this.set('headers', { 'Private-Token': 'sVfZzagWtemrV-suxYK-' });
```

Comment:
This shouldn't be here?

Reply:
This was mentioned a month ago at https://github.com/coala/git-task-list/pull/44#discussion_r194419949